src/project.ts

```diff
@@ -226,4 +256,41 @@ export class ProjectModule {
     public deleteProject(projectId: string) {
         return this.xhr.del(`/gdc/projects/${projectId}`);
     }
+
+    /**
+     * Gets aggregated feature flags for given project and current user
+     *
+     * @method getFeatureFlags
+     * @param {String} projectId - A project identifier
+     * @return {IFeatureFlags} Hash table of feature flags and theirs values where feature flag is as key
+     */
+    public getFeatureFlags(projectId: string): Promise<IFeatureFlags> {
+        return this.xhr.get(`/gdc/app/projects/${projectId}/featureFlags`)
+            .then((apiResponse: ApiResponse) => {
+                if (!apiResponse.response.ok) {
+                    throw new ApiResponseError(
+                        apiResponse.response.statusText,
+                        apiResponse.response,
+                        apiResponse.responseBody
+                    );
+                }
+                return apiResponse.getData();
+            })
+            .then((result: IFeatureFlagsResponse) => {
+                if (result && result.featureFlags) {
```

Comment:
are those checks necessary ? as based on interface type those attributes are not optional so we can count on them :)